Directory.Build.props

```diff
@@ -285,6 +288,8 @@
 
     <!-- Rhel 6 and FreeBSD doesn't support the source control git package so disable SourceLink -->
     <EnableSourceLink Condition="$(RuntimeOS.StartsWith('rhel.6')) OR '$(_runtimeOSFamily)' == 'FreeBSD'">false</EnableSourceLink>
+    <EnableSourceLink Condition="'$(HostArch)' == 'Arm' OR '$(HostArch)' == 'Arm64'">false</EnableSourceLink>
```

Comment:
Does sourcelink need a bug to enable this scenario?

Reply:
Also, why use HostArch here instead of ArchGroup? You already mapped HostArch to ArchGroup

Reply:
This is because SourceLink ( microsoft.build.tasks.git) drags in native dependencies. When you for example run on x64 and ArchGroup=arm everything works ok. But let say you target x64 and you build on arm, that it fails. So it is not about what you building for but what you building on. 
As far as SourceLink: It may be nice if SourceLink supports ARM but there are other issues preventing full build on ARM @eerhardt . Asking them for ARM may complicate their build quite a bit and so far I did not find any limitations of setting this to false. For now, my primary use case is building single assembly using msbuild command and running tests. 
Rest os my tasks, I can do easily elsewhere. 

(and I added note about sourcelink on arm)

Reply:
I've opened https://github.com/dotnet/sourcelink/issues/231 to have the discussion if enabling source link on ARM makes sense.